### PR TITLE
Create AKS.cs

### LINKED ISSUE #2111: AKS primality test
- [x] C
- [x] C++
- [x] C# - @Yashank18
- [ ] CoffeeScript
- [ ] Dart - @Rukmini-Meda
- [ ] Go
- [x] Java
- [x] JS
- [ ] Kotlin
- [ ] Php
- [x] Python
- [x] Readme - @goyalmuskan
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript - @Kuljeet-123

### PR DESCRIPTION
**Pull request by Yashank18**
Fixes #2111 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implementation of AKS

#### Languages Used:
- C#

#### Files Added:
- AKS.cs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Yashank18** [comment]
@jainaman224  @SakshiShreya @singh-shreya6

File: AKS_Primarility_Test/AKS.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    
+        //Prints all the primes less than 100000000
+        
+        for(int i=0;i<1000000;i++)
+        {
+            if(isprm(i))
+            {
+                Console.Write(i);
+            }
+    }
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Fix indentation.

File: AKS_Primarility_Test/AKS.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
+using System;
+
+class AKS{
+    public static void main(String[] args){
+    
+        //Prints all the primes less than 100000000
+        
+        for(int i=0;i<1000000;i++)
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Add spaces around operator like `=`, `<`.

File: AKS_Primarility_Test/AKS.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+using System;
+
+class AKS{
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Add spaces before operator like `{`, `;`.

File: AKS_Primarility_Test/AKS.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            {
+                if(n%1==0)
+                    return false;
+                    i +=w;
+                    w=6-w;
+                    
+            }
+            return true;
+    }
+}
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Add new line and sample input output.

File: AKS_Primarility_Test/AKS.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    {   long i=5,w=2;
+        if(n==2 || n==3)
+        return true;
+        if(n%2 ==0 ||n%3==0||n==1)
+            return false;
+            while(i*i<=n)
+            {
+                if(n%1==0)
+                    return false;
+                    i +=w;
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Fix indentation.

**Comment by Yashank18** [comment]
Updated Please Check
@jainaman224

File: AKS_Primarility_Test/AKS.cs [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+If n ≤ r, output prime.
+For a = 2 to sqrt(phi (r))*log(2)(n)  
+    if (X+a)n≠ Xn+a (mod Xr − 1,n), output composite;
+Output prime.
+*/
+
+    
+    public  static boolean isprm(long n)
+    {   
+        long i,w=2;
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Add spaces around operators like `=`, `==`, `+` etc.

File: AKS_Primarility_Test/AKS.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
+using System;
+
+class AKS {
+    public static void main(String[] args){
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Add space before `{`.

File: AKS_Primarility_Test/AKS.cs [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    
+        //Prints all the primes less than 100000000
+        
+        for(int i = 0; i < 1000000; i++)
+        {
```
Comment (jainaman224) [inline AKS_Primarility_Test/AKS.cs]: Should have same template for blocks.
```
function() {
```

OR

```
function()
{
```

Throughout the file.

**Pull request merged**